force_bdss/mco/base_mco.py

```diff
@@ -79,11 +79,12 @@ def notify_new_point(self, optimal_point, optimal_kpis, weights):
             A list of weight values from 0.0 to 1.0 that have been assigned
             for this point to each KPI.
         """
-        self.notify(MCOProgressEvent(
-            optimal_point=optimal_point,
-            optimal_kpis=optimal_kpis,
-            weights=weights,
-        ))
+        self.notify(
+            WeightedMCOProgressEvent(
```

Comment:
Can this be an `MCOProgressEvent` for simplicity? I understand this will be removed _soon_, but still.